src/content-script.ts

```diff
@@ -4,11 +4,29 @@ import Mustache from "mustache";
 import { createElement } from "./elements";
 import { isBodyWithReplies, isRepositoryDetails } from "./validations";
 
-(async function main() {
+runMainOnLocationMatch();
+
+document.addEventListener("soft-nav:end", () => {
+	runMainOnLocationMatch();
+});
+
+function runMainOnLocationMatch() {
+	if (/\/.+\/.+\/(issues|pull)\/.+/.test(window.location.pathname)) {
+		main().catch((error) => {
+			console.error("Oh no!", error);
+		});
+	}
+}
+
+async function main() {
+	const openSavedRepliesButton = document.getElementById(
+		"saved-reply-new_comment_field"
+	);
+
 	// 1. Is this an issue I can reply to?
 	// (if not, exit from the page)
-	const replySummary = document.getElementById("saved-reply-new_comment_field");
-	if (!replySummary) {
+	if (!openSavedRepliesButton) {
+		console.error("Couldn't find clicker");
```

Comment:
[Bug] Now that `main()` runs on URLs like `https://github.com/JoshuaKGoldberg/refined-saved-replies/pull/156/files`, it's going to give a lot more false positives. That URL has one. I'm also getting this console error by:

1. Visiting https://github.com/JoshuaKGoldberg/refined-saved-replies/pull/156
2. Soft navigating to https://github.com/JoshuaKGoldberg/refined-saved-replies/pull/156/files

I think it'd be fine to not log an error so aggressively anymore - given that the extension is now able to run on URLs that don't necessarily have a comment field. What do you think?

Reply:
interesting point out!

Gitub can have custom path which will complicate the regex. I think I have been defeated by regex  

I'm thinking of a regex like `[a-zA-Z0-9-_]+` but this is an appropriate time to show me how it's done haha 😊

Reply:
Heh yeah regular expressions are annoying. Instead of getting all fancy, I'd say let's just match URLs that might not include a clicker, and remove the `console.error`.